src/Sentry/SentryOptions.cs

```diff
@@ -776,10 +776,14 @@ public StackTraceMode StackTraceMode
     public Func<bool>? CrashedLastRun { get; set; }
 
     /// <summary>
-    /// Keep <see cref="AggregateException"/> in sentry logging.
-    /// The default behaviour is to only log <see cref="AggregateException.InnerExceptions"/> and not include the root <see cref="AggregateException"/>.
-    /// Set KeepAggregateException to true to include the root <see cref="AggregateException"/>.
+    /// This property is no longer used.  It will be removed in a future version.
     /// </summary>
+    /// <remarks>
+    /// All exceptions are now sent to Sentry, including <see cref="AggregateException"/>s.
+    /// The issue grouping rules in Sentry have been updated to accomodate "exception groups",
+    /// such as <see cref="AggregateException"/> in .NET.
+    /// </remarks>
+    [Obsolete("This property is no longer used.  It will be removed in a future version.")]
```

Comment:
To we have an obsoletion policy? Like "remove in major versions only". Would it be obsoleted with an error first, then removed, or removed?

Reply:
Obsoleted like this, then left there until next major version (4.0.0).

If there were an equavlience, we'd connect it. But in this case there isn't any.